kernel/kernel-api/src/main/java/io/delta/kernel/internal/util/SystemClock.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (2023) The Delta Lake Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.delta.kernel.internal.util;
+
+/**
+ * A clock backed by the actual time from the OS as reported by the `System` API.
+ */
+public class SystemClock implements Clock {
```

Comment:
this can be a lamda

Reply:
I got an error message: "Unnamed classes are not supported at language level '8'"?